jacklemkus/jacklemkus/spiders/jacklemkus_spider.py

```diff
@@ -0,0 +1,102 @@
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.linkextractors import LinkExtractor
+
+
+class ItemSpider(CrawlSpider):
+    name = 'jacklemkus'
+    allowed_domains = ['jacklemkus.com']
+    start_urls = ['https://www.jacklemkus.com']
+
+    rules = [
+        Rule(LinkExtractor(allow=r'^https://www.jacklemkus.com/(\w+$|(\w+-\w+$))', restrict_css='#nav'), callback='parse_pagination'),
```

Comment:
let's update this RE.
Also make variable for restrict_css named `listing_css` for `navigation` and `product_css` for `products pages`